src/angular/styles/includes/_mixins.scss

```diff
@@ -164,6 +164,53 @@
   }
 }
 
+@mixin horizontalBorderAndShadowAsBoxShadow($attached: below, $borderWidth: 2, $scalingFactor: 1) {
+  $sign: 1;
+  @if $attached == above {
+    $sign: -1;
+  }
+  box-shadow: boxShadow($sign, $borderWidth, $scalingFactor),
+    horizontalBorderBoxShadow(-$sign, $borderWidth, $scalingFactor);
+}
+
+@function boxShadow($sign, $borderWidth, $scalingFactor) {
```

Comment:
Is this function necessary?

Reply:
will be used in usermenu and contextmenu